test/runtests.jl

```diff
@@ -88,6 +88,27 @@ prob = SteadyStateProblem(ODEProblem(fiip, u0, tspan))
 sol2 = solve(prob, DynamicSS(Tsit5(), abstol = 1e-4))
 @test typeof(u0) == typeof(sol2.u)
 
+for mode in (:default, :norm, :rel, :rel_norm, :rel_safe, :rel_safe_best, :abs, :abs_norm,
```

Comment:
these should be enums? It seems that's what the EnumX would've been for?

Reply:
Done